saleor/graphql/checkout/tests/test_checkout_lines.py

```diff
@@ -91,6 +91,24 @@ def test_checkout_lines_add_existing_variant(user_api_client, checkout_with_item
     assert line.quantity == 10
 
 
+def test_checkout_lines_add_existing_variant_over_allowed_stock(
+    user_api_client, checkout_with_item
+):
+    checkout = checkout_with_item
+    line = checkout.lines.first()
+    variant_id = graphene.Node.to_global_id("ProductVariant", line.variant.pk)
+
+    variables = {
+        "token": checkout.token,
+        "lines": [{"variantId": variant_id, "quantity": 9}],
```

Comment:
Some comments would be helpful somewhere in this test to describe the assumptions, e.g.:
- we have X quantity of the variant in stock
- we have Y quantity of this variant in checkout
- we try to add Z quantity to checkout, which should throw an error since Z + Y > X

Maybe instead of relying on stock quantities from fixtures, we could explicitly create a stock item at the beginning of the test, so that it never fails if someone ever changed the quantity in the fixture?